usr/lib/console-login-helper-messages/libutil.sh

```diff
@@ -34,7 +34,7 @@ write_via_tempfile() {
     local staged_file="$(mktemp --tmpdir="${tempfile_dir}" "${tempfile_template}")"
     cat > "${staged_file}"
     chmod a+r "${staged_file}"
-    ${mv_Z} "${staged_file}" "${generated_file}"
+    ${restorecon} "${staged_file}"
```

Comment:
we still need the `mv` command to move the file into place, but we now need a second command to fixup the SELinux labels.

```suggestion
    mv "${staged_file}" "${generated_file}"
    ${restorecon} "${generated_file}"
```